src/main/java/uk/gov/pay/api/model/directdebit/agreement/GetAgreementResponse.java

```diff
@@ -64,7 +64,7 @@ public String getReturnUrl() {
 
     @ApiModelProperty(value = "agreement state", required = true, example = "CREATED")
```

Comment:
"mandate state"

Reply:
i think this should be covered in https://payments-platform.atlassian.net/browse/PP-5299